airflow/contrib/hooks/gcp_vision_hook.py

```diff
@@ -354,3 +307,140 @@ def _get_autogenerated_id(response):
         if '/' not in name:
             raise AirflowException('Unable to get id from name... [{}]'.format(name))
         return name.rsplit('/', 1)[1]
+
+    @GoogleCloudBaseHook.catch_http_exception
+    @GoogleCloudBaseHook.fallback_to_default_project_id
+    def create_reference_image(
+        self,
+        location,
+        product_id,
+        reference_image,
+        reference_image_id=None,
+        project_id=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+    ):
+        """
+        For the documentation see:
+        :py:class:`~airflow.contrib.operators.gcp_vision_operator.CloudVisionReferenceImageCreateOperator`
+        """
+        client = self.get_conn()
+        self.log.info('Creating ReferenceImage')
+        parent = ProductSearchClient.product_path(project=project_id, location=location, product=product_id)
+
+        response = client.create_reference_image(
+            parent=parent,
+            reference_image=reference_image,
+            reference_image_id=reference_image_id,
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+
+        self.log.info('ReferenceImage created: %s', response.name if response else '')
+        self.log.debug('ReferenceImage created:\n%s', response)
+
+        if not reference_image_id:
+            # Refernece image  id was generated by the API
+            reference_image_id = self._get_autogenerated_id(response)
+            self.log.info('Extracted autogenerated ReferenceImage ID from the response: %s', product_set_id)
+
+        return reference_image_id
+
+    @GoogleCloudBaseHook.catch_http_exception
+    @GoogleCloudBaseHook.fallback_to_default_project_id
+    def delete_reference_image(
```

Comment:
According to the documentation of your library, these methods do not return any data. Hook does the same. I updated the operators so that they would not return anything.